### ISSUE TITLE
Step 02 - suggestion

### ISSUE THREAD

**Issue by elegarpes** [mozdevs/html5-games-workshop#9]
For beginners, maybe it is good to add that every time you refresh your game, you should clean browser cache too (at least, for me it is not enough to press cmd + r, I need cmd + shift + r). 

But it is just a personal suggestion :)

**Comment by pchiwan** [comment]
True! I had my Chrome dev tools open all the time for this not to happen... Good catch @elegarpes!

**Comment by belen-albeza** [comment]
We definitely need to add a "How to clear the cache and why" section.
